frontend/libs/studio-components/src/components/StudioTableRemotePagination/StudioTableRemotePagination.tsx

```diff
@@ -47,10 +51,21 @@ export const StudioTableRemotePagination = forwardRef<
   StudioTableRemotePaginationProps
 >(
   (
-    { columns, rows, size = 'medium', emptyTableFallback, onSortClick, pagination },
+    {
+      columns,
+      rows,
+      size = 'medium',
+      isLoading = false,
+      loadingText,
+      emptyTableFallback,
+      onSortClick,
+      pagination,
+    },
     ref,
   ): React.ReactElement => {
     const selectId = useId();
+    const tableBodyRef = React.useRef<HTMLTableSectionElement>(null);
```

Comment:
Can we import instead= 😄 
```suggestion
    const tableBodyRef = useRef<HTMLTableSectionElement>(null);
```

Reply:
Fixed 😊